src/templates/project.js

```diff
@@ -114,23 +136,18 @@ export default ({ data }) => {
                   <a href={`mailto:${Contact_email}`}>{Contact_email}</a>
                 </p>
               </div>
-
             </div>
-
-          </div>
-          <div className="col-xs-12 col-sm-4">
-
           </div>
+          <div className="col-xs-12 col-sm-4" />
         </div>
-
       </section>
     </Layout>
-  )
-}
+  );
+};
 
 export const pageQuery = graphql`
-  query ProjectQuery($slug: String) {
-    airtable(table: {eq: "projects"}, fields: {slug: {eq: $slug }}) {
+  query($name: String) {
+    airtable(table: { eq: "projects" }, data: { Project_Name: { eq: $name } }) {
```

Comment:
Update the query in the page template to use match where the data for Project_Name equals $name.